try{
    console.log(a[1])
}
catch(err){
    console.log(err)
}
console.log("i≈ülem bitti");